src/Exceptions/HashFailedException.php

```diff
@@ -2,7 +2,10 @@
 
 namespace Dpc\HashVerifier\Exceptions;
 
-
+/**
+ * Class HashFailedException
+ * @package Dpc\HashVerifier\Exceptions
```

Comment:
this isn't needed. It is understood from the namespace

Reply:
ok ill remove it